### PR TITLE
Shorten host property in network statsbeat

### PR DESCRIPTION
**Pull request by hectorhdzg**

### CODE REVIEW

**Comment by heyams** [comment]
can you add some tests for the regular expression? 
It's very similar to mine.

File: AutoCollection/Statsbeat.ts [hunk 1]
```diff
@@ -208,16 +208,27 @@ class Statsbeat {
     }
 
     private _getNetworkStatsbeatCounter(endpoint: number, host: string): Network.NetworkStatsbeat {
+        let shortHost = host;
+        try {
+            let hostRegex = new RegExp(/^https?:\/\/(?:www\\.)?([^\/.]+)/);
```
Comment (MSNev) [inline AutoCollection/Statsbeat.ts:213]: This looks like you have an extra \\, probably should be 
`/^https?:\/\/(?:www\.)?([^\/.]+)/`

File: Tests/AutoCollection/Statsbeat.tests.ts [hunk 2]
```diff
@@ -275,8 +275,8 @@ describe("AutoCollection/Statsbeat", () => {
         it("Multiple network categories and endpoints", (done) => {
             statsBeat.enable(true);
             const sendStub = sandbox.stub(statsBeat, "_sendStatsbeats");
-            statsBeat.countRequest(0, "breezeFirstEndpoint", 100, true);
-            statsBeat.countRequest(1, "quickpulseEndpoint", 200, true);
+            statsBeat.countRequest(0, "https://breezeFirstEndpoint.something.com", 100, true);
+            statsBeat.countRequest(1, "http://quickpulseEndpoint.something.com", 200, true);
```
Comment (MSNev) [inline Tests/AutoCollection/Statsbeat.tests.ts:279]: add a test with www. prefix that way it highlights the issue above with the extra `\` after the `www` in the regex

File: AutoCollection/Statsbeat.ts (hunk 1 above)
Comment (heyams) [reply AutoCollection/Statsbeat.ts:213]: java version: `^https?://(?:www\.)?([^/.-]+)`
it's missing a dash at the end [^\/.-]

westus02-1 or westus02-4 => send westus02 and we don't care about -1 or -4.

**Review by MSNev — APPROVED** [review]

**Pull request merged**